src/components/LoadingSkeletons/LoadingSkeleton.tsx

```diff
@@ -36,29 +36,36 @@ export interface ILoadingSkeletonProps extends IStandardSkeleton {
 	numRows?: number;
 	/** Controls if LoadingSkeleton replicated in number of columns. Default = 1. */
 	numColumns?: number;
+
```

Comment:
These margins allow easy setting of spacing between skeletons replicated on multiple rows and columns as well as single rendered skeleton

Reply:
would the `?` imply undefined - do we need to specify it explicitly in the type union?